### ISSUE TITLE
Option `:description-only:` ?

### ISSUE THREAD

**Issue by lgritz** [michaeljones/breathe#443]
I often find myself wanting to insert into my sphinx document the Doxygen *comments only* for some entity (function, group, namespace, class) -- with no "heading" and no listing of the individual functions. Is there a way to do that?

As an example, let's say the C++ looks like this (I'm going to use a group as an example here):

    /// @defgroup G
    /// @{
    /// Lengthy description of group G.
    
    /// Description of function f.
    int f();

    /// @}

And the Sphinx code looks like this:

    My text text text.

    .. doxygengroup:: G

    More of my text text.

This will render like this:

    My text text text.

    group G
        Lengthy description of group G.

        Functions
            int f()
                Description of function f.

    More of my text text.

But that's not what I always want. Instead, suppose that on the Sphinx side, if I say

    My text text text.

    .. doxygengroup:: G
        :description-only:

    More of my text text.

Then I want this "description-only" to cause it to render like this:

    My text text text.

    Lengthy description of group G.

    More of my text text.
   
See? With no listing for `f`, and for that matter, no "group G" heading and indentation.

Is there any way to accomplish this?

**Comment by rw1nkler** [comment]
I think this might also be a useful feature for displaying the `@file` description without all the elements defined in the file.
Currently, this is impossible using the `doxygenfile` directive.

**Comment by vermeeren** [comment]
Something related for the `doxygengroup` directive was merged recently with #501 in the form of a `sections` option. This could be added to other directives and perhaps satisfies this use case?

**Comment by alephzero** [comment]
@rw1nkler you should be able to display just the `@file` description with
```rst
  .. doxygenfile:: nutshell.h
     :sections: detaileddescription
```

**Comment by rw1nkler** [comment]
@vermeeren @alephzero Thank you very much! I need to check the `sections` option!

Maybe it will be good to regenerate the official documentation because the information about the mentioned option is missing there: https://breathe.readthedocs.io/en/latest/directives.html#id9. Unfortunately, the last build was created 1.5 years ago.

This option have a really nice description in the docs, which I regenerated locally.

**Comment by vermeeren** [comment]
@rw1nkler Yeah there is some bug with RTD which somehow stopped updating quite a while ago, see #475. I have no experience with RTD and also don't know who set this up (it was before I became the maintainer).

I am really considering just dropping the RTD docs and hosting it on a webserver I manage instead, as I don't like all of the complexity with third party hosted docs and prefer dumb nginx or similar.

**Comment by rw1nkler** [comment]
@vermeeren, First of all, you need maintainer rights to be able to set more advanced things related to documentation. Currently, there is only one RTD maintainer - [michaeljones](https://readthedocs.org/profiles/michaeljones/): 
https://readthedocs.org/projects/breathe/

However, a simple configuration can be done, without those rights, using RTD config file:
https://docs.readthedocs.io/en/latest/config-file/v2.html

I have created the basic one. I don't know exactly what kind of problems you have with the documentation building.
Preview of my local fork can be found here - [[PREVIEW]](https://rw1nkler-breathe.readthedocs.io/en/fix_rtd/). The config is [[HERE]](https://github.com/rw1nkler/breathe/blob/fix_RTD/.readthedocs.yml).

**Comment by vermeeren** [comment]
@rw1nkler Seems like it would be easy to fix in that case, thanks a lot for the info.

@michaeljones can you have a look at things considering you are the maintainer of the RTD project right now?
